src/lib/isPrimeNumber.js

```diff
@@ -0,0 +1,10 @@
+import assertString from './util/assertString';
+
+export default function isPrimeNumber(n) {
```

Comment:
This can be optimized a bit further - There's example code available on the [Primality test](https://en.wikipedia.org/wiki/Primality_test) wikipedia page with javascript code there and an explanation about the optimization.

It's worth documenting as well that this shouldn't be used with large prime numbers such as what's commonly needed in cryptography (nor should it be used for cryptography in general I'd suspect) because it's likely to operate slower than something like the Miller-Rabin primality test which is [used by openssl](https://github.com/openssl/openssl/blob/3ee3c4d2abeeeadc8d33498d1466a3a8381d286b/crypto/bn/bn_prime.c#L260).